trunner/harness/plo.py

```diff
@@ -146,14 +146,24 @@ def erase(self, device: str, offset: int, size: int, timeout: Optional[int] = No
             e.cmd = cmd
             raise e
 
-    def jffs2(self, device: str, erase: bool, cleanmarkers: PloJffs2CleanmarkerSpec, timeout: Optional[int] = None):
+    def jffs2(self, device: str, erase: bool, cleanmarkers: PloJffs2CleanmarkerSpec, timeout: int):
         """Performs jffs2 command."""
 
+        block_count = cleanmarkers.number_of_blocks
+
         cmd = f"jffs2 -d {device} -c {cleanmarkers}"
         if erase:
             cmd += " -e"
 
-        self.cmd(cmd, timeout)
+        self.send_cmd(cmd)
+
+        for i in range(0, block_count):
+            try:
+                self.dut.expect(f"(jffs2: block {i}/{block_count})", timeout=timeout) == 0
```

Comment:
`== 0` does nothing here (exception won't be raised if different),

also: I think we don't need grouping (`(...)`) in pattern, actually we're not using regexps here so `expect_exact` would be a better fit (as it's faster)